main.py

```diff
@@ -53,14 +53,14 @@ def __init__(self, *args, **kwargs) -> None:
 
     async def on_ready(self) -> None:
         if self.first_on_ready:
-            self.pool = await asyncpg.create_pool(
-                database=os.getenv("DATABASE_NAME", "chaotic"),
-                host="127.0.0.1", min_size=int(os.getenv("POOL_MIN", "20")),
-                max_size=int(os.getenv("POOL_MAX", "100")),
-                user=os.getenv("DATABASE_USER"),
-                password=os.getenv("DATABASE_PASSWORD"),
-            )
-            self.change_status.start()
+            # self.pool = await asyncpg.create_pool(
+            #     database=os.getenv("DATABASE_NAME", "chaotic"),
+            #     host="127.0.0.1", min_size=int(os.getenv("POOL_MIN", "20")),
+            #     max_size=int(os.getenv("POOL_MAX", "100")),
+            #     user=os.getenv("DATABASE_USER"),
+            #     password=os.getenv("DATABASE_PASSWORD"),
+            # )
+            # self.change_status.start()
```

Comment:
Database keeping will be decided in #81, not here, we should leave it uncommented until that.
Same goes for the line 74